packages/shared-components/src/customComponents/components/Fodselsnummer.jsx

```diff
@@ -68,10 +68,8 @@ export default class Fodselsnummer extends TextFieldComponent {
     const fnrTekst = fnrTekstWithMiddleSpace.replace(" ", "");
 
     if (!erGyldigeKontrollsifre(fnrTekst)) {
-      //translate based on key in validering file.
-      return this.t("fodselsnummerDNummer") === "fodselsnummerDNummer"
-        ? TEXTS.validering.fodselsnummerDNummer
-        : this.t("fodselsnummerDNummer");
+      // TODO translate based on key in validering file
+      return TEXTS.validering.fodselsnummerDNummer;
```

Comment:
Hadde det funket å sjekke om this.t("fodselsnummerDNummer) er type string, og returnere TEXTS.validering.fodselsnummerDNummer? Siden det virker som at feilen er sporadisk.